templates/shared/_blog-card.html

```diff
@@ -1,50 +1,102 @@
-
```

Comment:
Looks like this was copied from the latest blogs section on the u.com homepage some time ago, but a lot of this isn't relevant here. We can probably delete this (since it doesn't look like any other page is using it anymore) and replace it with something similar to [this](https://github.com/canonical/ubuntu.com/blob/main/templates/download/shared/_latest-blogs_download.html)

Reply:
I tried to replace it but it compromises a bit on flexibility like for e.g. "Section class" parameter allows the current card to be placed even in between the content(though that might be less of the cases).
May be we can re-visist it later and see what relevant additions/removals could be done to this.